src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Computer"
+        class="computer"
+      />
+
+      <div class="text">
+        <h1 class="text__big">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the page, so you need to use the another tag here and fix class
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina
          <br />
          5K Monoblock (MNED2UA/A)
        </h2>
```